src/getAppSyncConfig.js

```diff
@@ -22,6 +28,16 @@ export default function getAppSyncConfig(context, appSyncConfig) {
       type: source.type,
     };
 
+    /**
+     * Returns the tableName resolving reference
+     */
+    const getTableName = (table) => {
+      if (table && table.Ref) {
+        return get(context.serverless.service, `resources.Resources.${table.Ref}.Properties.TableName`);
```

Comment:
@marcoreni  Awesome. I was planning on finding a solution for this. Didn't expect it to be so simple 🎉 
Can we maybe throw an exception if `Ref` is not found?

Reply:
I digged into dynamodb-local and the process of handling the TableProperties was quite complex, but after a quick search the [Serverless tutorial](https://serverless.com/blog/writing-serverless-plugins-2/#which-table) pointed me in the right direction :D 

Added the exception as requested